src/OrbitGl/GlSlider.cpp

```diff
@@ -21,22 +21,25 @@ namespace orbit_gl {
 const float GlSlider::kGradientFactor = 0.25f;
 
 void GlSlider::OnMouseEnter() {
-  is_mouse_over_ = true;
+  CaptureViewElement::OnMouseEnter();
   if (QGuiApplication::instance() != nullptr) {
     QGuiApplication::setOverrideCursor(Qt::ArrowCursor);
   }
+  RequestUpdate(RequestUpdateScope::kDraw);
 }
 
 void GlSlider::OnMouseLeave() {
-  is_mouse_over_ = false;
+  CaptureViewElement::OnMouseLeave();
   if (QGuiApplication::instance() != nullptr) {
     QGuiApplication::restoreOverrideCursor();
   }
+  RequestUpdate(RequestUpdateScope::kDraw);
 }
 
-void GlSlider::OnMouseMove(int x, int y) {
+void GlSlider::OnMouseMove(const Vec2& mouse_pos) {
```

Comment:
I just realized: The sliders should always `RequestUpdate(Draw)` when the mouse position changed. The appearance of the slider is different depending on whether the mouse if over the scrollbar or the background, so this isn't fully covered by mouse enter / leave.